functions.php

```diff
@@ -26,13 +26,16 @@ function displayArtworks(array $data) : string
     }
     $return = '';
     foreach ($data as $artwork){
-        $return .= '<div class="artworks">';
-        $return .= '<img src="' . $artwork['image'] . '" height="250" />';
-        $return .= '<h2>Title: ' . $artwork['title'] . '</h2>';
-        $return .= '<h3>Artist: ' . $artwork['artist'] . '</h3>';
-        $return .= '<h4>Movement: ' . $artwork['movement'] . '</h4>';
-        $return .= '<h4>Year: ' . $artwork['year'] . '</h4>';
-        $return .= '</div>';
+        if (!$artwork['deleted']) {
```

Comment:
Would suggest filtering out the delete artworks with your select statement, so adding a where condition to make sure deleted is 0